ivy/functional/backends/paddle/experimental/activations.py

```diff
@@ -95,8 +111,17 @@ def silu(x: paddle.Tensor, /, *, out: Optional[paddle.Tensor] = None) -> paddle.
     return F.silu(x.cast("float32")).cast(x.dtype)
 
 
+@with_unsupported_device_and_dtypes(
+    {"2.5.1 and below": {"cpu": ("float16", "bfloat16", "complex64", "complex128")}},
+    backend_version,
+)
```

Comment:
The paddle backend's `expm1` function already supports complex numbers, and there shouldn't be anything else in the `jax_like` function which doesn't, so you can remove them from here.
```suggestion
@with_unsupported_device_and_dtypes(
    {"2.5.1 and below": {"cpu": ("float16", "bfloat16")}},
    backend_version,
)
```
Also, the `if paddle.is_complex(x): ...` section of the function below can be removed since it's handled in the `jax_like` function